src/Yasumi/Provider/Austria/LowerAustria.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2019 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <me@sachatelgenhof.com>
+ */
+
+namespace Yasumi\Provider\Austria;
+
+use DateTime;
+use Yasumi\Exception\InvalidDateException;
+use Yasumi\Exception\UnknownLocaleException;
+use Yasumi\Holiday;
+use Yasumi\Provider\Austria;
+
+/**
+ * Provider for all holidays in Lower Austria (Austria).
+ *
+ * @link https://en.wikipedia.org/wiki/Lower_Austria
+ */
+class LowerAustria extends Austria
+{
+    /**
+     * Code to identify this Holiday Provider. Typically this is the ISO3166 code corresponding to the respective
+     * country or sub-region.
+     */
+    public const ID = 'AT-3';
+
+    /**
+     * Initialize holidays for Lower Austria (Austria).
+     *
+     * @throws InvalidDateException
+     * @throws \InvalidArgumentException
+     * @throws UnknownLocaleException
+     * @throws \Exception
+     */
+    public function initialize(): void
+    {
+        parent::initialize();
+
+        // Add custom holidays.
+        $this->calculateStLeopoldsDay();
+    }
+
+    /**
+     * Saint Leopold's Day.
+     *
+     * Saint Leopold III, known as Leopold the Good, was the Margrave of Austria
+     * from 1095 to his death in 1136. He was a member of the House of
+     * Babenberg. He was canonized on 6 January 1485 and became the patron saint
+     * of Austria, Lower Austria, Upper Austria, and Vienna. His feast day is 15
+     * November.
+     *
+     * @link https://en.wikipedia.org/wiki/Leopold_III,_Margrave_of_Austria
+     *
+     * @throws InvalidDateException
+     * @throws \InvalidArgumentException
+     * @throws UnknownLocaleException
+     * @throws \Exception
+     */
+    private function calculateStLeopoldsDay(): void
```

Comment:
Can you add a condition for when this holiday was first introduced/started? If not know, I would then use St. Leopold's birthday/day of death as a condition. (Guessing that before his birth this wasn't a holiday). 
This will prevent this holiday to be calculated before his birth.